### PR TITLE
fix(protocol-designer): Fix flex deck view position in overview

### PR DESCRIPTION
**Pull request by koji**
# Overview
Fix flex deck view position in overview

centralize Flex deck view in Overview page 

Screenshot 2024-11-04 at 1 41 49 PM

add max-height to block deck view from enlarging
this is not an ideal way to display deck map but at this point, Mel agreed with this way.
thread https://opentrons.slack.com/archives/C07E9T2NAQK/p1730764305465249?thread_ts=1730738177.182339&cid=C07E9T2NAQK

https://github.com/user-attachments/assets/daccc044-e312-4628-a5af-9cac519c9bf3

close RQA-3333

## Changelog
low

### CODE REVIEW

**Comment by koji** [comment]
- [ ] max-height?

File: components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx [hunk 1]
```diff
@@ -41,10 +41,10 @@ export function RobotCoordinateSpaceWithRef(
         viewBoxOriginY + PADDING * 5
       } ${deckXDimension + PADDING * 2} ${deckYDimension - PADDING * 10}`
     } else {
-      const PADDING = 20
-      wholeDeckViewBox = `${viewBoxOriginX - PADDING} ${
-        viewBoxOriginY + PADDING
-      } ${deckXDimension + PADDING * 2} ${deckYDimension + PADDING * 2}`
+      const PADDING = 10
```
Comment (ncdiehl11) [inline components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx]: I know this isn't introduced here, but should we pull the const declaration out of the if/else block and into a ternary above? Not sure best practice in JS.

File: protocol-designer/src/pages/Designer/DeckSetup/DeckSetupContainer.tsx [hunk 2]
```diff
@@ -179,6 +179,7 @@ export function DeckSetupContainer(props: DeckSetupTabType): JSX.Element {
         height={zoomIn.slot != null ? '75vh' : '70vh'}
         flexDirection={DIRECTION_COLUMN}
         padding={SPACING.spacing40}
+        maxHeight="39.375rem" // this is to block deck view from enlarging
```
Comment (ncdiehl11) [inline protocol-designer/src/pages/Designer/DeckSetup/DeckSetupContainer.tsx:182]: 👍

**Review by ncdiehl11 — APPROVED** [review]
Left a style comment but looks good!

**Comment by jerader** [comment]
I had a thought, since when the screen is smaller, the slot details is not shown, could we have it appear underneath or above the deck? 

Screenshot 2024-11-05 at 09 56 52

**Comment by koji** [comment]
> I had a thought, since when the screen is smaller, the slot details is not shown, could we have it appear underneath or above the deck?
> 
> Screenshot 2024-11-05 at 09 56 52

I think we need the design team's input. The design team may not notice that.

File: components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx (hunk 1 above)
Comment (koji) [reply components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx]: for performance, I think there wouldn't be a difference, but I will use your suggestion for readability.
thanks!

**Comment by koji** [comment]
Seems that slot detail issue will require to fix responsiveness first.
I will merge this into edge.

**Pull request merged**